src-java/base-topology/base-storm-topology/src/test/java/org/openkilda/wfm/share/mappers/HistoryMapperTest.java

```diff
@@ -0,0 +1,157 @@
+                .build();
+        flowDumpPayload = FlowDumpPayload.builder()
+                .affinityGroupId(AFFINITY_GROUP_ID)
+                .allocateProtectedPath(ALLOCATE_PROTECTED_PATH)
+                .bandwidth(BANDWIDTH)
+                .destinationInnerVlan(DESTINATION_INNER_VLAN)
+                .destinationPort(DESTINATION_PORT)
+                .destinationSwitch(DESTINATION_SWITCH)
+                .destinationVlan(DESTINATION_VLAN)
+                .diverseGroupId(DIVERSE_GROUP_ID)
+                .encapsulationType(FLOW_ENCAPSULATION_TYPE)
+                .forwardCookie(FORWARD_COOKIE)
+                .forwardMeterId(FORWARD_METER_ID)
+                .forwardPath(FORWARD_PATH)
+                .forwardStatus(FORWARD_STATUS)
+                .ignoreBandwidth(IGNORE_BANDWIDTH)
+                .loopSwitchId(LOOP_SWITCH_ID)
+                .maxLatency(MAX_LATENCY)
+                .pathComputationStrategy(PATH_COMPUTATION_STRATEGY)
+                .periodicPings(PERIODIC_PINGS)
+                .pinned(PINNED)
+                .reverseCookie(REVERSE_COOKIE)
+                .reverseMeterId(REVERSE_METER_ID)
+                .reversePath(REVERSE_PATH)
+                .reverseStatus(REVERSE_STATUS)
+                .sourceInnerVlan(SOURCE_INNER_VLAN)
+                .sourcePort(SOURCE_PORT)
+                .sourceSwitch(SOURCE_SWITCH)
+                .sourceVlan(SOURCE_VLAN)
+                .type(TYPE)
+                .build();
+    }
+
+    @Test
+    public void toFlowHistoryPayloadTest() {
+        FlowHistoryPayload payload = INSTANCE.map(action);
+
+        assertEquals(payload.getTimestamp(), action.getTimestamp().getEpochSecond());
+        assertEquals(payload.getAction(), action.getAction());
+        assertEquals(payload.getDetails(), action.getDetails());
+        assertEquals(payload.getTimestampIso(), action.getTimestamp().atOffset(ZoneOffset.UTC).toString());
+    }
+
+    @Test
+    public void toFlowHistoryEntryTest() {
+        FlowHistoryPayload flowHistoryPayload = INSTANCE.map(action);
+
+        ArrayList<FlowHistoryPayload> flowHistoryPayloads = new ArrayList<>();
+        flowHistoryPayloads.add(flowHistoryPayload);
+
+        ArrayList<FlowDumpPayload> flowDumpPayloads = new ArrayList<>();
+        flowDumpPayloads.add(flowDumpPayload);
+
+        FlowHistoryEntry entry = INSTANCE.map(event, flowHistoryPayloads, flowDumpPayloads);
+
+        assertEquals(entry.getAction(), action.getAction());
+        assertEquals(entry.getDetails(), action.getDetails());
+        assertEquals(entry.getTimestamp(), event.getTimestamp().getEpochSecond());
+        assertEquals(entry.getFlowId(), event.getFlowId());
+        assertEquals(entry.getActor(), event.getActor());
+        assertEquals(entry.getDumps(), flowDumpPayloads);
+        assertEquals(entry.getPayload(), flowHistoryPayloads);
+        assertEquals(entry.getTimestampIso(), event.getTimestamp().atOffset(ZoneOffset.UTC).toString());
```

Comment:
Also could you pleas add one mere assert.Just to be sure that time is human readable 
```
assertEquals(entry.getTimestampIso(), "2022-04-21T06:41:41.292Z");
```

Reply:
Okay, done